ci/infra/openstack/output.tf

```diff
@@ -1,19 +1,15 @@
-output "hostnames_masters" {
-  value = "${openstack_dns_recordset_v2.master.*.name}"
-}
-
 output "ip_masters" {
-  value = ["${openstack_networking_floatingip_v2.master_ext.*.address}"]
+  value = "${zipmap(openstack_compute_instance_v2.master.*.name, openstack_networking_floatingip_v2.master_ext.*.address)}"
 }
 
 output "ip_workers" {
-  value = ["${openstack_networking_floatingip_v2.worker_ext.*.address}"]
+  value = "${zipmap(openstack_compute_instance_v2.worker.*.name, openstack_networking_floatingip_v2.worker_ext.*.address)}}"
 }
 
 output "ip_internal_load_balancer" {
   value = "${openstack_lb_loadbalancer_v2.lb.vip_address}"
 }
 
 output "ip_load_balancer" {
-  value = "${openstack_networking_floatingip_v2.lb_ext.address}"
+  value = "${zipmap([openstack_lb_loadbalancer_v2.lb.name], [openstack_networking_floatingip_v2.lb_ext.address])}"
```

Comment:
output "ip_load_balancer" {
  value = "${map(openstack_lb_loadbalancer_v2.lb.name, openstack_networking_floatingip_v2.lb_ext.address)}"
}

Since openstack lb is string not list, you need to have map instead of zipmap.

Reply:
My mistake. You made to list. My bad.